### ISSUE TITLE
Pack deb packages

### ISSUE THREAD

**Issue by cblp** [ff-notes/ff#115]
for Ubuntu

**Comment by willbasky** [comment]
should it be packed when master build successfully on CI and handy?

**Comment by cblp** [comment]
Yes, CI must test that the package is built properly, and the release package must be exported.
